calico/common.py

```diff
@@ -497,7 +547,21 @@ def validate_rules(profile_id, rules):
                 if tag is None:
                     continue
                 if not VALID_ID_RE.match(tag):
-                    issues.append("Invalid %s '%r'." % (tag_type, tag))
+                    issues.append("Invalid %s %r." % (tag_type, tag))
```

Comment:
Extreme nit: I'd add a colon after %s.

(I wasn't going to say that, but then I noticed that you did this in the corresponding selector code below :-) )